src/panel/common/commonActions.js

```diff
@@ -54,10 +76,27 @@ function isInvalidTags(tagArr) {
 	return false;
 }
 
+function isInvalidID(id) {
+	// Checks whether id is a number and an integer
+	const numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+	for (var i = 0; i < id.length; i++) {
+		if (numbers.indexOf(id[i]) == -1) {
+			indicateInvalidID();
+			return true;
```

Comment:
The issue with this is that it accepts `1.` as valid, which the server then rejects